submission/programs/adae.R

```diff
@@ -0,0 +1,283 @@
+# Name: ADAE
+#
+# Label: Adverse Event Analysis Dataset
+#
+# Input: ae, adsl
+# Output: adae.xpt
+#
+# Developed by : [Author/Developer name]
+
+library(admiral)
+library(dplyr)
+library(tidyr)
+library(lubridate)
+library(stringr)
+library(diffdf)
+library(xportr)
+library(metacore)
+library(metatools)
+
+# ----------------------------------------------------------------------------#
```

Comment:
We should agree on a consistent style of programming and commenting

Reply:
Agree in principle but I'd say let's try to merge this and the other remaining ADaMs as soon as possible and then do a final "clean up" of all files before submitting.

Reply:
Let me know which way we should follow.

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.